docs/build-instructions/windows.md

```diff
@@ -21,57 +21,47 @@ You can use either:
 Whichever version you use, ensure that:
 
  * The default installation folder is chosen so the build tools can find it
- * If using Visual Studio make sure Visual C++ support is selected/installed
- * If using Visual C++ Build Tools make sure Windows 8 SDK is selected/installed
+ * Visual C++ support is selected/installed if using Visual Studio
+ * Windows 8 SDK is selected/installed if using Visual C++ Build Tools
  * A `git` command is in your path
- * Set the `GYP_MSVS_VERSION` environment variable to the Visual Studio/Build Tools version (`2013` or `2015`) e.g. ``[Environment]::SetEnvironmentVariable("GYP_MSVS_VERSION", "2015", "User")`` in PowerShell or set it in Windows advanced system settings control panel.
+ * The `GYP_MSVS_VERSION` environment variable is set to the corresponding Visual Studio/Build Tools version (`2013` or `2015`)
```

Comment:
Let's keep the command-line syntax of how to set environment variables - SETX is the equivalent in command-line. It also means you don't have to reopen you command prompt as you do with the UI.